pom.xml

```diff
@@ -405,12 +388,22 @@ http://www.w3.org/2001/XMLSchema-instance">
                     <aggregate>true</aggregate>
                     <instrumentation>
                         <excludes>
-                            <exclude>com/intuit/wasabi/tests/**/*.class</exclude>
+                            <exclude>com/intuit/wasabi/**/*Test.class</exclude>
```

Comment:
This probably won't exclude all functional tests. the /tests/** was there to exclude those.